app/src/main/res/layout/weather_frag.xml

```diff
@@ -0,0 +1,74 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:background="@color/teal_700"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <TextView
+            android:id="@+id/date"
+            style="@style/BaseTextStyle"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/icon_weather"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+
+        <ImageView
+            android:id="@+id/icon_weather"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/real_feel"
+            app:layout_constraintStart_toEndOf="@+id/date"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_width="@dimen/icon_width"
+            android:layout_height="@dimen/icon_height" />
+
+        <TextView
+            android:id="@+id/actual_temp"
+            style="@style/ActualTempStyle"
+            app:layout_constraintEnd_toEndOf="@id/real_feel"
+            app:layout_constraintStart_toStartOf="@id/real_feel"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+
+        <TextView
+            android:id="@+id/real_feel"
+            style="@style/Sensation"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/temp_min"
+            app:layout_constraintStart_toEndOf="@+id/icon_weather"
+            app:layout_constraintTop_toBottomOf="@id/actual_temp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+
+        <TextView
+            android:id="@+id/temp_max"
+            style="@style/BaseTextStyle"
+            app:layout_constraintBottom_toTopOf="@+id/temp_min"
+            app:layout_constraintEnd_toEndOf="@+id/temp_min"
+            app:layout_constraintStart_toStartOf="@+id/temp_min"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+             />
+
+        <TextView
+            android:id="@+id/temp_min"
+            style="@style/BaseTextStyle"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/real_feel"
+            app:layout_constraintTop_toBottomOf="@id/temp_max"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+             />
```

Comment:
pls put this end of the component on line 70

Reply:
Done thx!